### ISSUE TITLE
Paste schema, to update or verify it (or verify JSON against a know schema)

### ISSUE THREAD

**Issue by bldesign** [jsonsystems/json-schema#88]
Hi,

in the previous version (not sure which), it was possible to past the schema (paste JSON Schema on the right side). This possibility should remain as it provides several benefits:

1. Check a previously made JSON Schema against a JSON
1. Update a previously made JSON Schema 

Is that possible?

If not, I am forced to re-edit the infered JSON Schema even if I have it offline already perfectly done.

Anyway, as said, we do appreciate your contribution and it's a very attractive app.

**Comment by jsonsystems** [comment]
@bldesign You are completely correct, this did exist until fairly recently. I removed it because I wans't happy with the implementation - it was too buggy and was rarely used. I do plan on reintroducing it later this year. I understand this will not help you right now though. 

Soon you will be able to login and save your schemas so if you have it once, you won't need to generate it from scratch.

I'll leave this open to track the implementation and for other users to add their thoughts.

Thank you for taking the time to add this issue.

**Comment by bldesign** [comment]
Good. Saving online on the cloud is not a bad idea. But it's not the same. I don't want my schema to be stored on the cloud. The only good one is on my VSC (private git) system, in production or in dev.

I would need the ability to paste my schema, paste my JSON and verify it. That will, for sure, help many others. If you tend to make your service cloud ready is good too, but it fulfills other needs, in my opinion. Can you restore paste Schema and paste JSON ability and add a Verify button to validate the JSON against the schema? That would be perfectly awesome 😎

**Comment by elnezah** [comment]
Agree with @bldesign on both of his messages.

**Comment by mrsureshkumar** [comment]
I didn't know about this limitation, I had done half the work, in between switched over to other schema, now I am unable to continue with my work with the first schema. Any work around?

**Comment by jsonsystems** [comment]
@mrsureshkumar To clarify, you mean you have generated a schema from JSON and then inferred a new schema, and now you would like to load the first schema you generated?

**Comment by mrsureshkumar** [comment]
I am working on many schema's and tried to load a previously generated schema to make few updates. When I add my JSON schema on the left side, it generates schema's schema. We need an option to copy n paste the existing JSON schema to modify/update. (Something like a radio option o JSON or o JSON Schema on the left side to select the type of document we are pasting)

**Comment by jsonsystems** [comment]
@mrsureshkumar Right, but was the previously generated scheam generated using www.jsonschema.net, or was ite generated outside of the tool?

**Comment by dgutman** [comment]
It seems like the ability to edit the schema in "code" mode was also disabled?  Do all edits/updates to the schema now need to be done using the edit interface.    I guess this would be disabled since you can no longer "paste"... but sometimes adding attributes is much quicker this way...

![image](https://user-images.githubusercontent.com/713166/92968627-7a4edc80-f449-11ea-8372-9581ae70d38a.png)

**Comment by dgutman** [comment]
So now that paste mode was disabled... I am having trouble figuring out how to do something VERY basic...

How do I add a top level key to an existing schema... I can see how to edit existings keys, but I don't see a button to add new ones..



![image](https://user-images.githubusercontent.com/713166/93095809-9634c700-f671-11ea-8f2f-d2a5777fb1ac.png)

**Comment by justlevine** [comment]
Has there been any progress on reinstating these features?
I can't import an existing schema or edit the code, and see no way to change the property names, nor add/delete any properties.

**Comment by JohanRyberg** [comment]
I would love to see this implemented
